azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/configuration/CallCompositeConfiguration.kt

```diff
@@ -17,6 +18,7 @@ internal class CallCompositeConfiguration {
     val remoteParticipantsConfiguration: RemoteParticipantsConfiguration = RemoteParticipantsConfiguration()
     var callScreenOrientation: CallCompositeSupportedScreenOrientation? = null
     var setupScreenOrientation: CallCompositeSupportedScreenOrientation? = null
+    var telecomOptions: CallCompositeTelecomOptions? = null
```

Comment:
If we can make non-nullable, a little better (with a default value or lateinit).